x/incentives/keeper/distribute_test.go

```diff
@@ -672,19 +686,17 @@ func (suite *KeeperTestSuite) TestDistributeConcentratedLiquidityInternal() {
 	for _, tc := range testCases {
 		suite.Run(tc.name, func() {
 			suite.SetupTest()
+			suite.Ctx = suite.Ctx.WithBlockTime(defaultBlockTime)
 			suite.PrepareConcentratedPool()
 
 			suite.FundAcc(tc.sender, sdk.NewCoins(sdk.NewCoin(defaultRewardDenom, sdk.NewInt(10000))))
-			gauges := suite.SetupGauges(tc.gauges, defaultRewardDenom)
+			gauges := suite.SetupGauges([]perpGaugeDesc{tc.gauge}, defaultRewardDenom)
 
-			for _, gauge := range gauges {
-				coins, err := suite.App.IncentivesKeeper.DistributeConcentratedLiquidityInternal(suite.Ctx, tc.poolId, tc.sender, tc.incentiveDenom, tc.incentiveAmount, tc.emissionRate, tc.startTime, tc.minUptime, gauge)
-				if tc.expectedError {
-					suite.Require().Error(err)
-				} else {
-					suite.Require().NoError(err)
-					suite.Require().Equal(tc.expectedCoins, coins)
-				}
+			err := suite.App.IncentivesKeeper.DistributeConcentratedLiquidity(suite.Ctx, tc.poolId, tc.sender, sdk.NewCoin(tc.incentiveDenom, tc.incentiveAmount), tc.emissionRate, tc.startTime, tc.minUptime, gauges[0])
```

Comment:
small nit: why test for only default gauge[0], why not for doubleLengthGauge as well?

Reply:
Why does it matter?

Reply:
just testing over different gauges, instead of just default one